code/modules/projectiles/projectile/bullets.dm

```diff
@@ -186,6 +187,28 @@
 		T.part = organ
 		LAZYADD(organ.implants, T)
 
+/obj/item/projectile/bullet/acid
+	name = "acid capsule"
+	damage = 20
+	damage_type = DAMAGE_BURN
+	agony = 15
+
+/obj/item/projectile/bullet/acid/on_hit(atom/target, blocked, def_zone)
+	. = ..()
+
+	var/splatter_dir = starting ? get_dir(starting, target.loc) : dir
+	new /obj/effect/temp_visual/dir_setting/bloodsplatter(target.loc, splatter_dir, COLOR_VIOLET)
+	playsound(loc, 'sound/weapons/sear.ogg', 60, TRUE)
+
+	if(istype(target, /mob/living/simple_animal/hostile/giant_spider))
+		var/mob/living/simple_animal/hostile/giant_spider/gremmy = target
+		gremmy.take_overall_damage(0, 100, src)
+		if(!gremmy.stat)
+			walk_away(gremmy, starting, 5, gremmy.move_to_delay)
```

Comment:
There should not be calls that alters the AI behavior of AI mobs outside the mob itself, the AI of a mob manages what the mob does and if something affects it, it should be relayed to the AI of the mob itself so that it can carry it out, as they can conflict with each other or create unexpected situations otherwise

If you want the gremorian to move away, you should add a function to it to signal it that it should move away, and alter the AI mob processing to take that request into consideration and carry it out